<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CopyFilterGUI</class>
 <widget class="QWidget" name="CopyFilterGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>488</width>
    <height>94</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="verticalSpacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="copyLabel">
     <property name="text">
      <string>Copy channels:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QTabWidget" name="tabWidget">
     <property name="tabPosition">
      <enum>QTabWidget::East</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="elideMode">
      <enum>Qt::ElideRight</enum>
     </property>
     <property name="usesScrollButtons">
      <bool>false</bool>
     </property>
     <property name="documentMode">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="icon">
       <iconset resource="../Editor.qrc">
        <normaloff>:/icons/applications-graphics-rotated.ico</normaloff>:/icons/applications-graphics-rotated.ico</iconset>
      </attribute>
      <attribute name="title">
       <string/>
      </attribute>
      <attribute name="toolTip">
       <string>Graph</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QGraphicsView" name="graphicsView">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="sizeAdjustPolicy">
          <enum>QAbstractScrollArea::AdjustToContents</enum>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
         <property name="renderHints">
          <set>QPainter::Antialiasing|QPainter::TextAntialiasing</set>
         </property>
         <property name="dragMode">
          <enum>QGraphicsView::RubberBandDrag</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="icon">
       <iconset resource="../Editor.qrc">
        <normaloff>:/icons/format-justify-fill-rotated.ico</normaloff>:/icons/format-justify-fill-rotated.ico</iconset>
      </attribute>
      <attribute name="title">
       <string/>
      </attribute>
      <attribute name="toolTip">
       <string>Assignments</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QScrollArea" name="scrollArea">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOn</enum>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="CopyFilterGUIForm" name="form">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>380</width>
            <height>90</height>
           </rect>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="spaceReserveLabel">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>Selected channels:</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CopyFilterGUIForm</class>
   <extends>QWidget</extends>
   <header>CopyFilterGUIForm.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../Editor.qrc"/>
 </resources>
 <connections/>
</ui>
